app/cdap/components/Administration/TetheringTabContent/OdfTetheringConnections/Connections/ConnectionsTable.tsx

```diff
@@ -15,12 +15,27 @@
  */
 
 import React from 'react';
+import { IConnection } from '../types';
+
+interface IConnectionsTableProps {
+  tableData: IConnection[];
+}
+
+const ConnectionsTable = ({ tableData }: IConnectionsTableProps) => {
+  const transformedTableData = tableData.map((req) => ({
+    requestTime: '00:00:00',
```

Comment:
Should this have a date component? Also, is there a timezone?

Reply:
I added a TODO here in the second commit that this needs to be properly updated once the backend server captures request time. This is more of a placeholder for now. Good point on the timezone. Need to clarify that.